pkg/asset/installconfig/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,160 @@
+package kubevirt
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/kubevirt"
+)
+
+// Platform collects kubevirt-specific configuration.
+func Platform() (*kubevirt.Platform, error) {
+	var (
+		kubeConfig, namespace, apiVIP, ingressVIP, networkName, storageClass string
+		err                                                                  error
+	)
+
+	if kubeConfig, err = selectKubeConfig(); err != nil {
+		return nil, err
+	}
+
+	if namespace, err = selectNamespace(); err != nil {
+		return nil, err
+	}
+
+	if apiVIP, err = selectAPIVIP(); err != nil {
+		return nil, err
+	}
+
+	if ingressVIP, err = selectIngressVIP(); err != nil {
+		return nil, err
+	}
+
+	if networkName, err = selectNetworkName(); err != nil {
+		return nil, err
+	}
+
+	if storageClass, err = selectStorageClass(); err != nil {
+		return nil, err
+	}
+
+	return &kubevirt.Platform{
+		InfraClusterKubeConfig:   kubeConfig,
+		InfraClusterNamespace:    namespace,
+		InfraClusterStorageClass: storageClass,
+		APIVIP:                   apiVIP,
+		IngressVIP:               ingressVIP,
+		NetworkName:              networkName,
+	}, nil
+}
+
+func selectKubeConfig() (string, error) {
+	var selectedKubeConfig string
+
+	defaultValue := ""
+	home := os.Getenv("HOME")
```

Comment:
In my opiniion you need to take the kubeconfig from a relative place in the running directory

Reply:
Done